tests/test_settings.py

```diff
@@ -27,7 +27,7 @@
 from pydantic.env_settings import SettingsError  # pylint: disable=no-name-in-module
 
 from fawltydeps.main import build_parser
-from fawltydeps.settings import Action, OutputFormat, Settings
+from fawltydeps.settings import Action, OutputFormat, Settings, default_ignore_unused
```

Comment:
```suggestion
from fawltydeps.settings import Action, OutputFormat, Settings, DEFAULT_IGNORE_UNUSED
```

Reply:
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/b6afa98689e4720130fad434d0ce13e2d2b44a26.